ansible/roles/keycloak/defaults/main.yml

```diff
@@ -57,11 +57,11 @@ wildfly_standalone_config_path: /opt/keycloak/keycloak/standalone/configuration/
 keycloak_ext:  <extension module="org.keycloak.keycloak-server-subsystem"/>
 #keycloak_subsystem:
 #keycloak_cache_container:
-#keycloak_default_ds: 
+#keycloak_default_ds:
```

Comment:
You shouldn't be changing any code from the role files. These seem to be default variables that you can override when invoking the role.